src/tasks/WasmAppBuilder/WasmAppBuilder.cs

```diff
@@ -23,6 +23,8 @@ public class WasmAppBuilder : Task
     [Required]
     public string? MainJS { get; set; }
 
+    public string? WasmEnableES6 { get; set; }
```

Comment:
Shouldn't it be a `bool?`?

Reply:
Somehow it didn't work well for me and it was not worth fighting it.